src/events/index.ts

```diff
@@ -1,2 +0,0 @@
-export {PlaylistEvents} from './events';
```

Comment:
what if instead of delete the file use:
`export * from './events';`

Reply:
i can check but anyway i thin the inner index.ts is indeed redundant